samples/nrf9160/lwm2m_client/src/sensors/env_sensor.c

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <drivers/sensor.h>
+#include <stdlib.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(env_sensor, CONFIG_APP_LOG_LEVEL);
+
+#if defined(CONFIG_ENV_SENSOR_USE_EXTERNAL)
+#define ENV_SENSOR_NODE DT_PATH(soc, peripheral_40000000, i2c_a000, bme680_76)
+#define ENV_SENSOR_DEV_LABEL DT_LABEL(ENV_SENSOR_NODE)
+#elif defined(CONFIG_ENV_SENSOR_USE_SIM)
+#define ENV_SENSOR_DEV_LABEL "SENSOR_SIM"
+#define GAS_RES_SIM_BASE CONFIG_ENV_SENSOR_GAS_RES_SIM_BASE
+#define GAS_RES_SIM_MAX_DIFF CONFIG_ENV_SENSOR_GAS_RES_SIM_MAX_DIFF
+#endif
+
+static const struct device *env_sensor_dev;
+
+bool initialised;
```

Comment:
Should this be at least _static_. And it has been used only inside one function so it could be placed inside of it.

Reply:
Removed completely, as this check wasn't necessary